docs/nuget-org/Deprecate-packages.md

```diff
@@ -61,3 +61,82 @@ Project `My.Test.Project` has the following deprecated packages
+
+For example, let's say I have two packages:
+
+* My deprecated legacy package, `Contoso.Legacy` with 3 million downloads
+* My latest package, `Contoso.Latest` with 5 downloads
+
+NuGet.org prefers search results with higher downloads/popularity. Given the search query "Contoso", my deprecated package `Contoso.Legacy` would likely rank above my latest package `Contoso.Latest` in search results.
+
+To solve this problem, I can apply to transfer the popularity of my deprecated legacy package to my latest package. This would cause `Contoso.Latest` to rank higher in search results, while `Contoso.Legacy` would rank lower. Only the internal popularity scores for the packages is impacted, the actual download count for each package will not be affected.
+
+> [!Note]
+> Popularity transfers can make it significantly harder for consumers to find the legacy package.
+
+See the table below to get a concrete idea of how a popularity transfer may impact search rankings for the query "Contoso":
+
+| Search ranking 	| Before popularity transfer     	| After popularity transfer      	|
+|----------------	|--------------------------------	|--------------------------------	|
+| 1              	| *Contoso.Legacy, 3M downloads*  	| *Contoso.Latest, 5 downloads* 	|
+| 2              	| Contoso.Scanner, 2M downloads  	| Contoso.Scanner, 2M downloads  	|
+| 3              	| Contoso.Core,  1.5M downloads  	| Contoso.Core,  1.5M downloads  	|
+| 4              	| Contoso.UI, 1M downloads       	| Contoso.UI, 1M downloads       	|
+| ...            	| ...                            	| ...                            	|
+| 20             	| *Contoso.Latest, 5 downloads* 	| *Contoso.Legacy, 3M downloads*   	|
+
+### Popularity transfer application process
+
+1. Review the [popularity transfer requirements](#popularity-transfer-requirements).
+2. Email [account@nuget.org](mailto:account@nuget.org) with the deprecated package whose popularity should be transferred, and, the list of stable package(s) that should receive the popularity transfer.
+
+After the application is submitted, we will notify you of your application's acceptance or rejection (with the criteria that caused rejection). We may need to ask additional identifying questions to confirm owner identity.
+
+#### Popularity transfer requirements
+
+* The legacy packages and new packages must share all owners.
+* The new packages must be clearly related to the legacy packages in naming and function (i.e. an evolution or next generation).
+* All versions of the legacy packages must be deprecated and point to the new packages receiving the transfer.
+* The popularity transfer must not cause confusion for NuGet users or worsen the NuGet search experience.
+* The new packages must have a stable version.
+* The legacy package must have fewer than 5 new packages.
```

Comment:
I think mentioning this int he Package splits section might be sufficient. It may be confusing and unnecessary for most.

Otherwise, I would propose an alternative wording: "The popularity of a single legacy package can be transferred to and divided among no more than 5 newer packages. See package splits for more details about this scenario."

Reply:
Agreed, I'll take it out as it is a corner case

Reply:
Removed